src/main/java/stream/flarebot/flarebot/commands/moderation/ReportsCommand.java

```diff
@@ -114,8 +115,17 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                         if (guild.getReportManager().getReport(id).getStatus().equals(status)) {
                             MessageUtils.sendInfoMessage("Current status is: **" + status.getMessage() + "**", channel, sender);
                         } else {
+                            ReportStatus old = guild.getReportManager().getReport(id).getStatus();
                             guild.getReportManager().getReport(id).setStatus(status);
                             MessageUtils.sendSuccessMessage(String.format("Changed status of Report with ID: **%d** to **%s**", id, status.getMessage()), channel, sender);
+                            EmbedBuilder eb = new EmbedBuilder();
+                            eb.setTitle("Report edited");
+                            eb.setColor(Color.WHITE);
+                            eb.addField("Report ID", String.valueOf(id), true);
+                            eb.addField("Old Status", old.getMessage(), true);
+                            eb.addField("New Status", guild.getReportManager().getReport(id).getStatus().getMessage(), true);
+                            eb.addField("Responsible moderator", sender.getAsMention(), true);
```

Comment:
Just chain these, I see no reason to make a new builder and keep doing this. You can easily new line each addField anyway as a chain. Builders are builders for a reason

Reply:
I don't see a good way to chain them. if you could send me what your thinking that whould be helpfull

Reply:
nvm I'm stupid